modules/service/src/main/java/com/devonfw/module/service/common/api/client/ServiceClientFactory.java

```diff
@@ -79,4 +79,23 @@
    */
   <S> AsyncServiceClient<S> createAsync(Class<S> serviceInterface, Map<String, String> config);
 
+  /**
+   * @param <S> type of the {@link com.devonfw.module.service.common.api.client.context.ServiceContext#getApi() service
+   *        API}.
+   * @param serviceInterface the {@link Class} reflecting the interface that defines the API of your {@link Service}.
+   * @param config the {@link Map} with explicit configuration properties. See
+   *        {@link com.devonfw.module.service.common.base.config.ServiceConfigProperties} for further details.
+   * @return an {@link AsyncServiceClient} allowing to call service operations asynchronously.
+   */
+
+  <S> SyncServiceClient<S> createSync(Class<S> serviceInterface, Map<String, String> config);
+
+  /**
+   * @param <S> type of the {@link com.devonfw.module.service.common.api.client.context.ServiceContext#getApi() service
+   *        API}.
+   * @param serviceInterface the {@link Class} reflecting the interface that defines the API of your {@link Service}.
+   * @return an {@link SyncServiceClient} allowing to call service operations synchronously.
+   */
+  <S> SyncServiceClient<S> createSync(Class<S> serviceInterface);
```

Comment:
We already have this API. For historic reasons the methods are called `create`. When async support was added the `createAsync` methods had been added.
I do not see why we have to add `createSync` additionally that would IMHO do the same as the `create` methods. Do I miss something?